jit/codegen_x86.cpp

```diff
@@ -4733,9 +4733,16 @@ LOWFUNC(NONE,NONE,2,raw_ftan_rr,(FW d, FR s))
 }
 LENDFUNC(NONE,NONE,2,raw_ftan_rr,(FW d, FR s))
 
+#ifdef CPU_x86_64
+#define REX64 emit_byte(0x48);
```

Comment:
the omission of the semicolon when calling the macro causes problems with 3rd party tools such as source code formatters, IDEs etc that do not implement a full C preprocessor.

suggested improval:
```c
#define REX64() emit_byte(0x48)
#else 
#define REX64()
```
and later
```REX64();```